debian/saml-soap-proxy/saml-soap-proxy.yml

```diff
@@ -0,0 +1,162 @@
+    - type: http
+      port: 50161
+
+  requestLog:
+    type: classic
+    appenders:
+      - type: file
+        currentLogFilename: /var/log/ida/debug/saml-soap-proxy_access.log
+        archivedLogFilenamePattern: /var/log/ida/debug/saml-soap-proxy_access.log.%d.gz
+        archivedFileCount: 7
+      - type: logstash-file
+        currentLogFilename: /var/log/ida/saml-soap-proxy_access.log
+        archivedLogFilenamePattern: /var/log/ida/saml-soap-proxy_access.log.%d.gz
+        archivedFileCount: 7
+      - type: logstash-syslog
+        facility: local2
+        tag: saml-soap-proxy
+
+saml:
+  entityId: https://signin.service.gov.uk
+  eidasEntityId: https://www.signin.service.gov.uk/SAML2/metadata/connector
+
+enableRetryTimeOutConnections: true
+
+httpClient:
+  timeout: 60s
+  timeToLive: 10m
+  connectionTimeout: 4s
+  retries: 3
+  keepAlive: 60s
+  chunkedEncodingEnabled: false
+  validateAfterInactivityPeriod: 5s
+  tls:
+    protocol: TLSv1.2
+    verifyHostname: false
+    trustStorePath: /ida/saml-soap-proxy/saml-soap-proxy.ks
+    trustStorePassword: ${KEYSTORE_PASSWORD}
+
+soapHttpClient:
+  timeout: 50s
+  timeToLive: 10m
+  connectionTimeout: 50s
+  keepAlive: 60s
+  chunkedEncodingEnabled: false
+  validateAfterInactivityPeriod: 5s
+  tls:
+    protocol: TLSv1.2
+    verifyHostname: true
+    trustStorePath: /ida/truststore/ida_root_ca_truststore.ts
+    trustStorePassword: ${KEYSTORE_PASSWORD}
+
+healthCheckSoapHttpClient:
+  timeout: 2s
+  timeToLive: 10m
+  connectionTimeout: 2s
+  keepAlive: 60s
+  chunkedEncodingEnabled: false
+  validateAfterInactivityPeriod: 5s
+  tls:
+    protocol: TLSv1.2
+    verifyHostname: true
+    trustStorePath: /ida/truststore/ida_root_ca_truststore.ts
+    trustStorePassword: ${KEYSTORE_PASSWORD}
+
+samlEngineUri: ${SAML_ENGINE_URL}
+
+configUri: ${CONFIG_URL}
+
+eventSinkUri: ${EVENT_SINK_URL}
+
+policyUri: ${POLICY_URL}
+
+metrics:
+  reporters:
+    - type: graphite
+      host: graphite
+      port: 2003
+      prefix: ${GRAPHITE_PREFIX}
+      frequency: 10s
+
+
+serviceInfo:
+  name: saml-soap-proxy
+
+matchingServiceExecutorConfiguration:
+  corePoolSize: 5
+  maxPoolSize: 10
+  keepAliveDuration: 10s
+
+rpTrustStoreConfiguration:
+  path: /ida/truststore/ida_rp_truststore.ts
+  password: ${KEYSTORE_PASSWORD}
+
+featureFlagConfiguration: {}
+
+logging:
+  level: INFO
+  appenders:
+    - type: file
+      currentLogFilename: /var/log/ida/debug/saml-soap-proxy.log
+      archivedLogFilenamePattern: /var/log/ida/debug/saml-soap-proxy.log.%d.gz
+      archivedFileCount: 7
+    - type: logstash-file
+      currentLogFilename: /var/log/ida/saml-soap-proxy.log
+      archivedLogFilenamePattern: /var/log/ida/saml-soap-proxy.log.%d.gz
+      archivedFileCount: 7
+    - type: logstash-syslog
+      facility: local2
+      tag: saml-soap-proxy
+    - type: raven
+      dsn: ${SENTRY_DSN:-naive+https://sentry/1}
+      threshold: ERROR
+      tags: service-name:saml-soap-proxy
+
+metadata:
```

Comment:
The signin.service of the uri and the expectedEntityId should also be environment variables.

Reply:
Fixed. I'd brought in a stale version of the config - whoops! Top spot!